etl/tests/test_transform.py

```diff
@@ -17,13 +18,24 @@ def azure_reader(populated_azure_emulator: transform.AzureCoordinates) -> transf
 
 def test_reader(local_reader: transform.LocalExtractReader) -> None:
     output = list(local_reader.extracted_rows())
+
     output = sorted(output, key=lambda row: row['BUILDER'])
     unique_builders = {row['BUILDER'] for row in output}
+
```

Comment:
did you intend to add these two returns here?

Reply:
Only kind of. Originally the test I added was just another assertion in this test, but I decided to factor it out into it's own dedicated test. When I was doing stuff in the other test I added some newlines to make it prettier/more readable to me.